src/models/request/character-request.ts

```diff
@@ -45,18 +44,18 @@ export class CharacterRequest extends Request<CharacterResult[]> {
             GROUP_CONCAT(DISTINCT ?consort; SEPARATOR=",") AS ?consorts
             GROUP_CONCAT(DISTINCT ?parents; SEPARATOR=",") AS ?parents
             WHERE {
-                <${selectedCharacterUri}> rdfs:label ?label ;
+                dbr:${selectedCharacterUri} rdfs:label ?label ;
                     dbo:abstract ?abstract .
     
-                OPTIONAL { <${selectedCharacterUri}> dbo:thumbnail ?thumbnail }
-                OPTIONAL { <${selectedCharacterUri}> dbp:abode ?abode }
-                OPTIONAL { <${selectedCharacterUri}> dbp:children ?children}
-                OPTIONAL { <${selectedCharacterUri}> dbp:consort ?consort }
-                OPTIONAL { <${selectedCharacterUri}> dbp:godOf ?godOf }
-                OPTIONAL { <${selectedCharacterUri}> dbp:parents ?parents }
-                OPTIONAL { <${selectedCharacterUri}> dbp:planet ?planet }
-                OPTIONAL { <${selectedCharacterUri}> dbp:symbol ?symbol }
-                OPTIONAL { <${selectedCharacterUri}> dbp:siblings ?siblings }
+                OPTIONAL { dbr:${selectedCharacterUri} dbo:thumbnail ?thumbnail }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:abode ?abode }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:children ?children}
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:consort ?consort }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:godOf ?godOf }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:parents ?parents }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:planet ?planet }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:symbol ?symbol }
+                OPTIONAL { dbr:${selectedCharacterUri} dbp:siblings ?siblings }
```

Comment:
look for fix in case of special characters

Reply:
changed